public/app/dashboard/splash.html

```diff
@@ -0,0 +1,11 @@
+<div class="leaderboard-list-container">
```

Comment:
@johncrossman Does this class work for the dashboard and leaderboard both? If so, might be worth renaming it to something more generic and moving into the main app CSS.